lib/classes/Service.js

```diff
@@ -52,6 +53,12 @@ class Service {
       .then((serverlessFileParam) => {
         const serverlessFile = serverlessFileParam;
         // basic service level validation
+        const ymlVersion = serverlessFile.frameworkVersion;
+        if (ymlVersion && ymlVersion !== version) {
+          const msgPartOne = `The Serverless version (${version}) is not matching the`;
+          const msgPartTwo = `"frameworkVersion" (${ymlVersion}) in serverless.yml`;
+          throw new SError(`${msgPartOne} ${msgPartTwo}`);
```

Comment:
Just a minor proposal / pointer.

We've used this pattern for multi-line error messages throughout the codebase:
https://github.com/serverless/serverless/blob/ca1131865518a67e69a31d63d5e51c26e3525228/lib/plugins/aws/deploy/compile/events/apiGateway/lib/validate.js#L17-L24